### ISSUE TITLE
Missing Contribution and Development Documentation

### ISSUE THREAD

**Issue by mattlknight** [libpnet/libpnet#364]
I am having a difficult time properly testing and validating my contirbutions to this repo. I believe some documentation might help this along. I would be willing to write this documentation as well, if I could get some help understanding the current build and test environment.

**Comment by JuxhinDB** [comment]
Hi @mattlknight -- are you still interesting in helping with this?

**Comment by mattlknight** [comment]
@JuxhinDB Yes, my apologies for the absence lately. I've been traveling for work a ton this summer and moving this weekend. Should be more involved in the next week or two.

**Comment by JuxhinDB** [comment]
That's great, enjoy the vacation. :-) -- whenever you would like to be involved let me know.
